syntax_and_semantics/capturing_blocks.md

```diff
@@ -2,7 +2,14 @@
 
 A block can be captured and turned into a `Proc`, which represents a block of code with an associated context: the closured data.
 
-To capture a block you must specify it as a method's block argument, give it a name and specify the input and output types. For example:
+To capture a block you must:
```

Comment:
I really dislike direct address in a formal specification. Could you rephrase this? For example here: `A block is captured if these conditions are met:` (list items should be in 3rd person)

Reply:
LMK what you think of the new wording.